src/chi/components/header/header.scss

```diff
@@ -222,10 +222,3 @@
     }
   }
 }
-
-/* IE11 CSS hack */
-@media all and (-ms-high-contrast:none) {
-  .o-header__content, .o-header__toolbar ~ .o-header__content {
-    height: 4rem;
```

Comment:
@davidojedacl removing this classes header is not looking good in IE11. We must make these classes bigger, via media queries, and ensuring the smallest scenarios are covered with the new heights.

<img width="964" alt="Screen Shot 2019-05-23 at 12 38 42 PM" src="https://user-images.githubusercontent.com/17043903/58246821-5d742e80-7d58-11e9-8a31-23c3f5395734.png">

<img width="980" alt="Screen Shot 2019-05-23 at 12 39 03 PM" src="https://user-images.githubusercontent.com/17043903/58246829-649b3c80-7d58-11e9-8ed4-280b87e012ca.png">